Makefile

```diff
@@ -72,6 +72,10 @@ link-git-hooks: ## Install git hooks
 fmt: ## Format changed go
 	@scripts/fmt.sh
 
+fmt-all: ### Format all go files with goimports and gofmt
+	find . -name "*.go" -not -path "./vendor/*" -exec gofmt -w "{}" \;
+	find . -name "*.go" -not -path "./vendor/*" -exec goimports -l -w "{}" \;
+
```

Comment:
we are running `gofmt` and `goImports` in `fmt`. I am not sure why we need this change.

Reply:
The `fmt` target only updates staged files. I can drop this one, if it's not needed. I used it to format all `go` files.

Reply:
if we plan to keep this it should also skip generated code like the mock package